src/dashboard/src/components/ingest/views.py

```diff
@@ -500,6 +573,86 @@ def transfer_backlog(request):
     return helpers.json_response(return_list)
 
 
+@decorators.elasticsearch_required()
+def transfer_appraisal(request):
+    """
+    AJAX endpoint to query for and return transfer backlog items in the appraisal tab format.
+    """
+    # Get search parameters from request
+    results = None
+    queries, ops, fields, types = advanced_search.search_parameter_prep(request)
+
+    if not 'query' in request.GET:
+        query = elasticSearchFunctions.MATCH_ALL_QUERY
+    else:
+        queries, ops, fields, types = advanced_search.search_parameter_prep(request)
+
+        try:
+            query = advanced_search.assemble_query(
+                queries,
+                ops,
+                fields,
+                types,
+                # Specify this as a filter, not a must_have, for performance,
+                # and so that it doesn't cause the "should" queries in a
+                # should-only query to be ignored.
+                filters={'term': {'status': 'backlog'}},
+            )
+        except:
+            logger.exception('Error accessing index.')
```

Comment:
Is this ERROR-level?  If not, then `logger.foo(..., exc_info=True)` might be better.